### PR TITLE
Additional Traitor Templates

### PR DESCRIPTION
**Pull request by Aether217**
## About The Pull Request
Adds 4 new templates to help antagonists who have issues with new ideas or pushing themselves out of their comfort zone. It also adds another objective to the Data Theft template so it's not just rushing the blackbox and sitting in a pod.

## Why It's Good For The Game
More templates, more fun. A common issue we often have is people wanting to have their antag status removed because they didn't prepare a gimmick. These 4 templates should give people a few more options, ranging from more serious and mild to more extreme and a bit silly even.

## Changelog
:cl:
add: Added 4 new templates: Mind Control Victim, Lone Pirate, Cyber Terrorist, Stamp Collector.
add: Added an extra objective to the Data Theft template
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/ambitions/code/ambition_templates.dm [hunk 1]
```diff
@@ -28,8 +28,8 @@
 /datum/ambition_template/data_theft
 	name = "Data Theft"
 	narrative = "You have been selected out of Donk. Co's leagues of potential sleeper operatives for one particular task."
-	objectives = list("Steal the Blackbox.", "Escape on the emergency shuttle alive and out of custody.")
-	tips = list("You should add an objective on how you plan to go about this task.", "The Blackbox is located in Telecomms on most Stations.", "The Blackbox doesn't fit in most bags. You'd need a bag of holding to hide it!")
+	objectives = list("Steal the Blackbox.", "steal at least 20 technology nodes from the research division." "Escape on the emergency shuttle alive and out of custody.")
+	tips = list("You should add an objective on how you plan to go about this task.", "The Blackbox is located in Telecomms on most Stations.", "The Blackbox doesn't fit in most bags. You'd need a bag of holding to hide it!", "To steal technology we'll have to print a disk from science and sneak our way to a research console and download nodes on the disk!)
```
Comment (tf-4) [inline modular_skyrat/modules/ambitions/code/ambition_templates.dm]:
```suggestion
	tips = list("You should add an objective on how you plan to go about this task.", "The Blackbox is located in Telecomms on most Stations.", "The Blackbox doesn't fit in most bags. You'd need a bag of holding to hide it!", "To steal technology we'll have to print a disk from science and sneak our way to a research console and download nodes on the disk!")
```

Missed a quotation closing heh

File: modular_skyrat/modules/ambitions/code/ambition_templates.dm [hunk 2]
```diff
@@ -28,8 +28,8 @@
 /datum/ambition_template/data_theft
 	name = "Data Theft"
 	narrative = "You have been selected out of Donk. Co's leagues of potential sleeper operatives for one particular task."
-	objectives = list("Steal the Blackbox.", "Escape on the emergency shuttle alive and out of custody.")
-	tips = list("You should add an objective on how you plan to go about this task.", "The Blackbox is located in Telecomms on most Stations.", "The Blackbox doesn't fit in most bags. You'd need a bag of holding to hide it!")
+	objectives = list("Steal the Blackbox.", "steal at least 20 technology nodes from the research division." "Escape on the emergency shuttle alive and out of custody.")
```
Comment (jjpark-kb) [inline modular_skyrat/modules/ambitions/code/ambition_templates.dm]:
```suggestion
	objectives = list("Steal the Blackbox.", "steal at least 20 technology nodes from the research division.", "Escape on the emergency shuttle alive and out of custody.")
```

File: modular_skyrat/modules/ambitions/code/ambition_templates.dm [hunk 3]
```diff
@@ -82,3 +82,28 @@
...
+	narrative = "I've made it in, my maties are watching and supplying me with gear. The vault, the pearl of the station, just waiting for someone to walk in and take it's treasure. But it's not just riches I'm after, the leaders of this ship have been tyrannical and I think it's time for some justice at sea."
+	objectives = list("Plan a raid on the vault and steal all the valuables inside, the golden belt especially will be a fine trophy.", "Capture a head of staff, maybe even the captain, and have the crew decide their faith, if they have been kind to their peope they will receive mercy.", "Escape alive and with your stolen goods.")
+	tips = list("While the valuables are for me the justice is not, I'm here to enforce the will of the crew and whatever they decide will be the outcome")
+	
+/datum/ambition_template/cyberterror
+	name = "Cyber Terrorist"
+	narative = "<%Uplink connection established, welcome #Agent#%> We have detected multiple ways to weaken the station through their artificial units, you will be tasked with manipulating the cybernetic elements of the station to greatly hinder it's productivity and teach them a lesson about over-reliance on technology!"
```
Comment (tf-4) [inline modular_skyrat/modules/ambitions/code/ambition_templates.dm]:
```suggestion
	narrative = "<%Uplink connection established, welcome #Agent#%> We have detected multiple ways to weaken the station through their artificial units, you will be tasked with manipulating the cybernetic elements of the station to greatly hinder it's productivity and teach them a lesson about over-reliance on technology!"
```

File: modular_skyrat/modules/ambitions/code/ambition_templates.dm [hunk 4]
```diff
@@ -82,3 +82,28 @@
...
 
 //TODO: Everything Past Cook and Scientist Ambitions
+
+/datum/ambition_template/mkultra
+	name = "Mind Control Victim"
+	narrative = "My head.... <groans> it keeps hurting, can't think straight... gotta get a grip. They've done something to me, I know it, it was supposed to just be another medical checkup but somethings wrong, I don't feel like I'm myself anymore."
+	objectives = list("Find out about the station's secret experiments, the documents hidden in the vault might have information on them.", "It happened after I went in for an appointment, perhaps interrogating the CMO will get me more information!", "They used some weird chamber on me, if I can get my hands on some of the disks they use to store technology perhaps I can expose them!", "I need to stay hidden though, they'll never believe me, or worse they'll sedate me and put me back in!")
+	tips = list("The workers, they probably don't know much about this so I should avoid harming them...", "The heads will likely be tough to crack but I know they're hiding it from me, sometimes you have to use force to get what you need", "There might be others like me, I should try to contact them.")
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/ambitions/code/ambition_templates.dm]:
```suggestion
	tips = list("The workers, they probably don't know much about this so I should avoid harming them...", "The heads will likely be tough to crack, but I know they're hiding it from me, sometimes you have to use force to get what you need.", "There might be others like me, I should try to contact them.")
```

File: modular_skyrat/modules/ambitions/code/ambition_templates.dm [hunk 5]
```diff
@@ -82,3 +82,28 @@
...
+	tips = list("A multitool can be used to mess with the focus of the camera, it also won't trigger it's alert system.", "With a bit of experimentation and some signallers we can remotely turn off power in areas via their APCs", "The ability to read all messages is quite powerful, shame that the Chief Engineer did not bother to hide the paper with the password in his office!")
+	
+/datum/ambition_template/stampman
+	name = "Stamp Collector"
+	narrative = "Stamps, such an underappreciated art! Did you ever stop to take a closer look at one before marking a piece of paper with it? If you did you'd notice all the intricate details that go into them, the beautiful sculpted pieces soaked in the ink... All left to gather dust in the offices of those moronic heads. I think it's time for someone with more appreciation for the art to take care of them!"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/ambitions/code/ambition_templates.dm]:
```suggestion
	narrative = "Stamps, such an underappreciated art! Did you ever stop to take a closer look at one before marking a piece of paper with it? If you did, you'd notice all the intricate details that go into them, the beautiful sculpted pieces soaked in the ink... All left to gather dust in the offices of those moronic heads. I think it's time for someone with more appreciation for the art to take care of them!"
```

File: modular_skyrat/modules/ambitions/code/ambition_templates.dm [hunk 6]
```diff
@@ -82,3 +82,28 @@
...
+	name = "Cyber Terrorist"
+	narative = "<%Uplink connection established, welcome #Agent#%> We have detected multiple ways to weaken the station through their artificial units, you will be tasked with manipulating the cybernetic elements of the station to greatly hinder it's productivity and teach them a lesson about over-reliance on technology!"
+	objectives = list("Sabotage the camera feed of the station, make it hard for them to see.", "Hack APCs in strategic locations, this way we can disable their machinery and lights as we see fit.", "Acquire the authentication code for telecommunications and use it to monitor or delete messages, perhaps we can acquire more information or aid another agent this way.", "Our gear allows us to subvert the station borgs, they will be of great use to us.")
+	tips = list("A multitool can be used to mess with the focus of the camera, it also won't trigger it's alert system.", "With a bit of experimentation and some signallers we can remotely turn off power in areas via their APCs", "The ability to read all messages is quite powerful, shame that the Chief Engineer did not bother to hide the paper with the password in his office!")
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/ambitions/code/ambition_templates.dm]:
```suggestion
	tips = list("A multitool can be used to mess with the focus of the camera, it also won't trigger it's alert system.", "With a bit of experimentation and some signallers, we can remotely turn off power in areas via their APCs.", "The ability to read all messages is quite powerful, shame that the Chief Engineer did not bother to hide the paper with the password in his office!")
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Just a few minor punctuation changes, otherwise this is pretty good, I'd say.

**Comment by Aether217** [comment]
Done with all of the changes above

**Comment by Lauwma** [comment]
Very good addition, good themes and good flavors

File: modular_skyrat/modules/ambitions/code/ambition_templates.dm [hunk 7]
```diff
@@ -28,8 +28,8 @@
 /datum/ambition_template/data_theft
 	name = "Data Theft"
 	narrative = "You have been selected out of Donk. Co's leagues of potential sleeper operatives for one particular task."
-	objectives = list("Steal the Blackbox.", "Escape on the emergency shuttle alive and out of custody.")
-	tips = list("You should add an objective on how you plan to go about this task.", "The Blackbox is located in Telecomms on most Stations.", "The Blackbox doesn't fit in most bags. You'd need a bag of holding to hide it!")
+	objectives = list("Steal the Blackbox.", "Steal at least 20 technology nodes from the research division.", "Escape on the emergency shuttle alive and out of custody.")
+	tips = list("You should add an objective on how you plan to go about this task.", "The Blackbox is located in Telecomms on most Stations.", "The Blackbox doesn't fit in most bags. You'd need a bag of holding to hide it!", "To steal technology we'll have to print a disk from science and sneak our way to a research console and download nodes on the disk!")
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/ambitions/code/ambition_templates.dm]:
```suggestion
	tips = list("You should add an objective on how you plan to go about this task.", "The Blackbox is located in Telecomms on most Stations.", "The Blackbox doesn't fit in most bags. You'd need a bag of holding to hide it!", "To steal technology we'll have to print a disk from science and sneak our way to a research console to download nodes on the disk!")
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Last little nitpick!

**Pull request merged**